go/vt/vttablet/tabletserver/vstreamer/vstreamer.go

```diff
@@ -148,7 +149,11 @@ func (vs *vstreamer) Stream() error {
 		return wrapError(err, vs.pos)
 	}
 
-	conn, err := binlog.NewSlaveConnection(vs.cp)
+	cp, err := dbconfigs.WithCredentials(vs.cp)
```

Comment:
I think this one is unnecessary, I looked at this last night...  NewSlaveConnection(..) calls connectForReplication(..), which already uses dbconfigs.WithCredentials.